lib/location/location_search_delegate.dart

```diff
@@ -65,7 +65,10 @@ class LocationSearchDelegate extends SearchDelegate<TrufiLocation> {
 
   @override
   Widget buildSuggestions(BuildContext context) {
+    final theme = Theme.of(context);
+
```

Comment:
why the theme is passed as parameter, I thought it should be part of the context? maybe this issues is more related with dialog build context issues

Reply:
https://github.com/trufi-association/trufi-core/blob/73b00220cbcf21cabdd7aff8f9e9855d2adf3a9f/lib/location/location_search_delegate.dart#L31 

The LoactionSearchDelegate defines the theme, background = white.

Reply:
so I think passing theme as parameter is not the good way, maybe we can override the LocationSearchDelegate background after used

Reply:
Yeah, I am also not happy with the solution. I tried it lower in the widget tree but as soon I get one level down it directly get overwritten from somewhere. The problem is that the SearchDelegate overwrites the Theme of the context and uses its own. 

Currently, I did not have a solution for that and actually did not wanted to invest so much time. But I can investigate where the Problem happens.